### PR TITLE
Add how i got here viz [WIP]

### PR DESCRIPTION
**Pull request by bsmithgall**
### What Changed
- Added dependencies for [d3](d3js.org) and [dagre-d3](https://github.com/cpettitt/dagre-d3)
- Added simple prototype dagre-d3 chart (attached below)
### Issues
- Fixes (eventually) first TODO in bottom of README
### TODO
- Add custom styles to highlight where each page is along the path
### Screenshots

screen shot 2015-02-07 at 6 12 14 pm

### CODE REVIEW

**Comment by mapsam** [comment]
:deciduous_tree:

File: _includes/javascripts.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<script src="{{ "/js/bower_components/lodash/dist/lodash.js" | prepend: site.baseurl }}"></script>
+<script src="{{ "/js/bower_components/graphlib/dist/graphlib.core.js" | prepend: site.baseurl }}"></script>
+<script src="{{ "/js/bower_components/dagre/dist/dagre.core.js" | prepend: site.baseurl }}"></script>
+<script src="{{ "/js/bower_components/dagre-d3/dist/dagre-d3.core.js" | prepend: site.baseurl }}"></script>
+<script src="{{ "/js/bower_components/d3/d3.js" | prepend: site.baseurl }}"></script>
```
Comment (bsmithgall) [inline _includes/javascripts.html:5]: TODO: Add a dependency to Grunt to build these into one central `vendor.js` file

**Comment by bsmithgall** [comment]
Because this project is turning into an actual thing, I'm closing this.

**Comment by mapsam** [comment]
:skull:

**Pull request closed without merging**